### DISCUSSION TITLE
Use ARIA with Copilot?

### DISCUSSION THREAD

**Discussion by goohst8** [lifan0127/ai-research-assistant#73] [Q&A] [answered]
Hi, is it possible to configure ARIA to use Copilot?

**Comment by lifan0127** [comment]
Hi @goohst8  Are you referring to Microsoft Copilot?

**Reply by goohst8** [reply]
Hi @lifan0127, that is exactly what I mean. Would I just need to change the URL in the options pane?

**Reply by lifan0127** [reply, marked as answer]
Hi @goohst8 ARIA requires access to the OpenAI APIs: https://platform.openai.com/. Here are the steps:
- Ceate an [OpenAI account](https://platform.openai.com/signup) or [sign in](https://platform.openai.com/login). 
- Navigate to the [API key page](https://platform.openai.com/account/api-keys) and "Create new secret key".
- Set up [billing information](https://platform.openai.com/account/billing/overview) so that you have access to GPT-4 models.

**Reply by goohst8** [reply]
Hi @lifan0127, thanks for the information. I was hoping I could set it up to use copilot as I have access to the pro version. Thanks again and this plug in is really cool.
